var/spack/repos/builtin/packages/py-pennylane/package.py

```diff
@@ -28,7 +40,23 @@ class PyPennylane(PythonPackage):
     depends_on("py-toml", type=("build", "run"))
     depends_on("py-appdirs", type=("build", "run"))
     depends_on("py-semantic-version@2.7:", type=("build", "run"))
-    depends_on("py-autoray@0.3.1:", type=("build", "run"))
+    depends_on("py-autoray", type=("build", "run"))
     depends_on("py-cachetools", type=("build", "run"))
-    depends_on("py-pennylane-lightning@0.28:", type=("build", "run"))
+    depends_on("py-pennylane-lightning", type=("build", "run"))
```

Comment:
Why were these changed? The lower bound seems correct from the setup.py

Reply:
I'll reintroduce these.